litmus-portal/graphql-server/pkg/handlers/playground_handler.go

```diff
@@ -0,0 +1,15 @@
+package handlers
```

Comment:
Should we name this pkg as graphql_server.go?

@amityt

Reply:
Yes, lets update it to graphql_server.go @namkyu1999